frodo.js

```diff
@@ -1,86 +1,83 @@
-const reversedString = string.split("").reverse().join("");
```

Comment:
SCRIPT-This new feature lacks unit tests. Could you add some to ensure everything works correctly?